client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py

```diff
@@ -0,0 +1,128 @@
+import pyblish.api
+
+from openpype.pipeline.publish import get_publish_repre_path
+
+
+class IntegrateShotgridVersion(pyblish.api.InstancePlugin):
+    """Integrate Shotgrid Version"""
+    order = pyblish.api.IntegratorOrder + 0.497
+    label = "Shotgrid Version"
+
+    def process(self, instance):
+        sg_session = instance.context.data["shotgridSession"]
+
+        # TODO: Use path template solver to build version code from settings
+        anatomy = instance.data.get("anatomyData", {})
+        version_name = "_".join(
+            [
+                anatomy["project"]["code"],
+                anatomy["parent"],
+                anatomy["asset"],
+            ]
+        )
+
+        if instance.data["shotgridTask"]:
+            version_name = "{0}_{1}".format(
+                version_name,
+                instance.data["shotgridTask"]["content"]
+            )
+
+        version_name = "{0}_{1}".format(
+            version_name,
+            "v{:03}".format(int(anatomy["version"]))
+        )
+
+        sg_version = self._find_existing_version(version_name, instance)
+
+        if not sg_version:
+            sg_version = self._create_version(version_name, instance)
+            self.log.info("Create Shotgrid version: {}".format(sg_version))
+        else:
+            self.log.info("Use existing Shotgrid version: {}".format(sg_version))
+
+        data_to_update = {}
+        intent = instance.context.data.get("intent")
```

Comment:
is this still supported in Ayon? I thought even in OP this had stopped working

Reply:
I think @iLLiCiTiT  might know the answer

Reply:
We will add it back.